src/ks2e-m-dc5-element.ts

```diff
@@ -0,0 +1,308 @@
+import { html, LitElement } from 'lit';
+import { customElement } from 'lit/decorators.js';
+import { ElementPin } from './pin';
+
+@customElement('wokwi-ks2e-m-dc5')
+export class KS2EMDC5Element extends LitElement {
+  y1pos = 8.85;
+  y2pos = 70.5;
+  x1pos = 14.7;
+  x2pos = 55;
+  x3pos = 95.3;
+  x4pos = 155.8;
+
+  get pinInfo(): ElementPin[] {
+    return [
+      { name: '1', x: this.x1pos, y: this.y1pos, signals: [], description: '1' },
+      { name: '4', x: this.x2pos, y: this.y1pos, signals: [], description: '4' },
+      { name: '6', x: this.x3pos, y: this.y1pos, signals: [], description: '6' },
+      {
+        name: '8',
+        x: this.x4pos,
+        y: this.y1pos,
+        signals: [{ type: 'power', signal: 'GND' }],
+        description: '8',
+      },
+      { name: '9', x: this.x1pos, y: this.y2pos, signals: [], description: '9' },
+      { name: '11', x: this.x2pos, y: this.y2pos, signals: [], description: '11' },
+      { name: '13', x: this.x3pos, y: this.y2pos, signals: [], description: '13' },
+      { name: '16', x: this.x4pos, y: this.y2pos, signals: [], description: '16' },
+    ];
+  }
+
+  render() {
+    return html`
+      <svg
+        width="44mm"
+        height="21mm"
```

Comment:
Looking at the [data sheet](https://www.datasheetq.com/datasheet-download/752684/1/KEPONT/KS2E-M-DC5), it seems like the size of the element is 21 x 10 x 12mm, so the width should be 21mm and the height should be 10mm. 

This probably means the smaller text ("DC 5V               1A/125V AC               2A/⠀30V DC") will be too small to read - so it's fine to remove it

Reply:
Also, changing the size means that the positions of the pins in pinInfo will have to be updated